docs/source/app-dev/bindings-java/quickstart.rst

```diff
@@ -122,28 +123,29 @@ In this section, you will run the quickstart application and get introduced to t
 
    .. _quickstart-sandbox:
 
-#. To run the :doc:`sandbox </tools/sandbox>` (a lightweight local version of the ledger), run ``daml sandbox --dar .daml/dist/quickstart-0.0.1.dar``
+#. Run ``daml sandbox --port 6865``
 
-   The output should look like this:
+#. In a separate terminal run the following:
 
-   .. code-block:: none
+- ``daml ledger upload-dar --host localhost --port 6865 .daml/dist/quickstart-0.0.1.dar``
 
-      Starting Canton sandbox.
-      Listening at port 6865
-      Uploading .daml/dist/quickstart-0.0.1.dar to localhost:6865
-      DAR upload succeeded.
-      Canton sandbox is ready.
+- ``daml script --ledger-host localhost --ledger-port 6865 --dar .daml/dist/quickstart-0.0.1.dar --script-name Main:initialize --output-file output.json``
 
+- ``daml navigator server localhost 6865 --port 7500``
 
-   The sandbox is now running, and you can access its :doc:`ledger API </app-dev/ledger-api>` on port ``6865``.
+To run the :doc:`sandbox </tools/sandbox>` (a lightweight local version of the ledger), run ``daml sandbox --dar .daml/dist/quickstart-0.0.1.dar``
 
-   .. _quickstart-script:
+#. Point your browser to http://localhost:7500 and log in as alice.
 
-#. Open a new terminal window and navigate to your project directory, ``quickstart``.
+#. Check that curl http://localhost:8080/iou returns:
+
+{"0":{"issuer":"EUR_Bank::NAMESPACE","owner":"Alice::NAMESPACE","currency":"EUR","amount":100.0000000000,"observers":[]}}
 
-#. To initialize the ledger with some parties and contracts we use :doc:`Daml Script </daml-script/index>` by running ``daml script --dar .daml/dist/quickstart-0.0.1.dar --script-name Main:initialize --ledger-host localhost --ledger-port 6865 --static-time``
 
-   .. _quickstart-navigator:
+
+   .. _quickstart-script:
```

Comment:
You probably either want to delete it or move it up to the line where you call `daml script`.